packages/shift-react-components/src/components/orders/order-single.js

```diff
@@ -61,7 +62,8 @@ export class OrderSingle extends PureComponent {
         },
         unit_quantity: lineItem.quantity,
         total: (lineItem.pricing.line_total_inc_tax / 100),
-        sub_total: (lineItem.pricing.line_total_inc_tax / 100)
+        sub_total: (lineItem.pricing.pre_discount_line_total_inc_tax / 100),
+        total_discount: (lineItem.pricing.line_discount_inc_tax / 100)
```

Comment:
should these be penceToPounds() calls(I'm just worried about handling floats)

Reply:
Did a little investigation and it turns out these are converted further down the line within the LineItem component